examples/tutorials/api/makers.py

```diff
@@ -342,3 +344,93 @@
 print(datasets)
 
 plt.show()
+
+######################################################################
+# Observation duration and effective livetime
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# It can often be useful to know the total number of hours spent
+# in the given field of view (without correcting for the acceptance
+# variation in the field of view. This can be computed using `make_observation_time_map`
+# as shown below
+#
+
+# Get the observations
+obs_id = data_store.obs_table["OBS_ID"][data_store.obs_table["OBJECT"] == "MSH 15-5-02"]
+observations = data_store.get_observations(obs_id)
+print("No. of observations: ", len(observations))
+
+# Define an energy range
+energy_min = 100 * u.GeV
+energy_max = 10.0 * u.TeV
+
+# define a offset cut (the camera field of view)
+offset_max = 2.5 * u.deg
+
+# define the geom
+source_pos = SkyCoord(228.32, -59.08, unit="deg")
+energy_axis_true = MapAxis.from_energy_bounds(
+    energy_min, energy_max, nbin=2, name="energy_true"
+)
+geom = WcsGeom.create(
+    skydir=source_pos,
+    binsz=0.02,
+    width=(6, 6),
+    frame="icrs",
+    proj="CAR",
+    axes=[energy_axis_true],
+)
+
+total_obstime = make_observation_time_map(observations, geom, offset_max=offset_max)
+
+
+plt.figure(figsize=(5, 5))
+ax = total_obstime.plot(add_cbar=True)
+# Add the pointing position on top
+for obs in observations:
+    ax.plot(
+        obs.get_pointing_icrs(obs.tmid).to_pixel(wcs=ax.wcs)[0],
+        obs.get_pointing_icrs(obs.tmid).to_pixel(wcs=ax.wcs)[1],
+        "+",
+        color="black",
+    )
+ax.set_title("Total observation time")
+plt.show()
+
+# However, since the acceptance of IACT cameras vary within the field of
+# view, what is often interesting is not simply the total number of
+# hours a source was observed, but the on-axis equivalent number of hours.
```

Comment:
```suggestion
# As the acceptance of IACT cameras vary within the field of
# view, it can also be interesting to plot the on-axis equivalent 
# number of hours.
```